.github/workflows/ci.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       if: runner.os == 'macOS'
       run: |
         brew update
-        brew install gcovr ninja
+        brew install gcovr ninja || true
```

Comment:
If these installations fail, is it safe to just continue?

If yes, why do we install them in the first place?

Reply:
https://github.com/SFML/SFML/pull/2499#issuecomment-1500960157

It's a workaround for this Homebrew issue that appeared out of nowhere. It still installs the packages we care about so we can't remove `brew install`.